### PR TITLE
Rr watsonx.data

### PR DESCRIPTION
**Pull request by r2raman**
Fixes | Relates to **[issue link]**

Content changes:

- Any content updates here

Code changes:

- Any code changes here

### CODE REVIEW

File: content/Watsonx/WatsonxData/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Watsonx.data
+toc: false
+---
+
+# IBM Watsonx.Data Technical Sales Workshop
+
+This workshop covers the IBM Watsonx.Data L3 badge lab, highlighting
```
Comment (ridgill) [inline content/Watsonx/WatsonxData/README.md:8]: Looks like we need to finish the sentence here.  "highlighting...."   

Also, let's use the all lowercase in this spot and above on line 6.

**Comment by ridgill** [comment]
I see changes to package-lock.json.  Wouldn't expect any updates there, but the reason is probably related to this comment on Jake's PR: https://github.com/ibm-build-lab/VAD-VAR-Workshop/pull/178#discussion_r1320187620

File: content/Watsonx/WatsonxData/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+
+
+## Prerequisites
+
+***coming soon***
+
+## Labs
+
+- **[Module 101: Environment Setup](/watsonx.data/101)**
```
[the remark this answers is not in the document]
Comment (r2raman) [reply content/Watsonx/WatsonxData/README.md]: fixed the path

**Review by ridgill — APPROVED** [review]
LGTM

**Pull request merged**